zipline/testing/fixtures.py

```diff
@@ -1540,7 +1540,7 @@ def make_adjustment_db_conn_str(cls):
         return cls.adjustments_db_path
 
 
-class WithSeededRandomPipelineEngine(WithTradingSessions, WithAssetFinder):
+class WithSeededRandomPipelineEngine(WithAssetFinder, WithTradingSessions):
```

Comment:
Is there a mro issue with `WithSeededRandomPipelineEngine`?

Reply:
No, there was and I fixed it, but according to @ssanderson `WithSeededRandomPipelineEngine` isn't even necessary here. I just needed to create a custom factor with no inputs.

Reply:
I don't fully understand the change of inheritance order, but if it isn't needed, should we undo the change?

Reply:
Sure, sorry my previous answer was pretty ambiguous because I wasn't actually talking about this specific place - there was an mro error when I had used `WithSeededRandomPipelineEngine` in my test code, but I no longer need it in my tests, so I will revert this change.